internal/cli/atlas/accesslists/create.go

```diff
@@ -134,12 +134,24 @@ func CreateBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create [entry]",
 		Short: "Create an IP access list for your project.",
+		Long:  `The access list can contain one or more trusted IP addresses, AWS security group IDs, and entries in Classless Inter-Domain Routing (CIDR) notation. You can create one access list per project. Note: the command does not overwrite existing entries in the access list. Instead, it adds the new entries to the list of entries.`,
```

Comment:
the cli can only create 1 access list at a time I worry `one or more` may be confusing here

Reply:
Edited, thank you!